pp/integrations/plugin-packs/procedures/operatingsystems-linux-centreon-monitoring-agent.md

```diff
@@ -0,0 +1,1362 @@
+| FILTERSTATE   | Filter process states (regexp can be used). You can use: 'zombie', 'dead', 'paging', 'stopped', 'InterrupibleSleep', 'running', 'UninterrupibleSleep' |                   |             |
+| WARNINGTIME   | Thresholds                                                                                                                                            |                   |             |
+| CRITICALTIME  | Thresholds                                                                                                                                            |                   |             |
+| WARNINGTOTAL  | Thresholds                                                                                                                                            |                   |             |
+| CRITICALTOTAL | Thresholds                                                                                                                                            |                   |             |
+| EXTRAOPTIONS  | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options).                                                    | --verbose         |             |
+
+</TabItem>
+<TabItem value="Swap" label="Swap">
+
+| Macro        | Description                                                                                        | Default value     | Mandatory   |
+|:-------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARNING      | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                    |                   |             |
+| CRITICAL     | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                    |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). |                   |             |
+
+</TabItem>
+<TabItem value="Systemd-Journal" label="Systemd-Journal">
+
+| Macro           | Description                                                                                                                                                     | Default value     | Mandatory   |
+|:----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| SINCE           | Defines the amount of time to look back at messages. Can beminutes (ie 5 "minutes ago") or 'cache' to use the timestamp from last execution. (default: 'cache') | cache             |             |
```

Comment:
```suggestion
| SINCE           | Defines the amount of time to look back at messages. Can beminutes (ie 5 "minutes ago") or 'cache' to use the timestamp from last execution | cache             |             |
```